diff --git a/Tools/msi/bundle/Default.wxl b/Tools/msi/bundle/Default.wxl
index d034afc..baee92e 100644
--- a/Tools/msi/bundle/Default.wxl
+++ b/Tools/msi/bundle/Default.wxl
@@ -89,9 +89,9 @@
   <String Id="ShortInstallLauncherAllUsersLabel">安装 py.exe 时使用管理员权限(&amp;L)</String>
   <String Id="ShortInstallLauncherBlockedLabel">Python 启动器已安装</String>
   <String Id="PrecompileLabel">预编译标准库(&amp;P)</String>
-  <String Id="Include_symbolsLabel">下载调试符号(&amp;S)</String>
-  <String Id="Include_debugLabel">下载调试二进制文件（需要 VS 2017 或更高版本）(&amp;G)</String>
-  <String Id="Include_freethreadedLabel">下载自由线程版本(&amp;F)</String>
+  <String Id="Include_symbolsLabel">安装调试符号(&amp;S)</String>
+  <String Id="Include_debugLabel">安装调试二进制文件（需要 VS 2026 或更高版本）(&amp;G)</String>
+  <String Id="Include_freethreadedLabel">安装自由线程版本(&amp;F)</String>
 
   <String Id="ProgressHeader">[ActionLikeInstallation]进度</String>
   <String Id="ProgressLabel">[ActionLikeInstalling]：</String>
@@ -120,9 +120,24 @@
   <String Id="FailureRestartText">必须重启计算机才能完成软件回滚。</String>
   <String Id="FailureRestartButton">重启(&amp;R)</String>
   <String Id="FailureExistingInstall">由于已存在安装，无法安装 [WixBundleName]。请使用“程序和功能”修改、修复或删除 [WixBundleName]。</String>
-  <String Id="FailureOldOS">安装 [WixBundleName] 至少需要 Windows 10 或 Windows Server 2016
+  <String Id="FailureWin7MissingSP1">安装 [WixBundleName] 需要 Windows 7 Service Pack 1 及所有适用更新。
 
-访问 &lt;a href="https://www.python.org/downloads/"&gt;python.org&lt;/a&gt; 下载更早版本的 Python。</String>
+请 &lt;a href="https://www.bing.com/search?q=how%20to%20install%20windows%207%20service%20pack%201"&gt;更新计算机&lt;/a&gt; 后重新安装。</String>
+  <String Id="FailureVistaMissingSP2">安装 [WixBundleName] 需要 Windows Vista Service Pack 2 及所有适用更新。
+
+请 &lt;a href="https://www.bing.com/search?q=how%20to%20install%20windows%20vista%20service%20pack%202"&gt;更新计算机&lt;/a&gt; 后重新安装。</String>
+  <String Id="FailureXPOrEarlier">安装和使用 [WixBundleName] 需要 Windows Vista 或更高版本。
+
+请访问 &lt;a href="https://www.python.org/"&gt;python.org&lt;/a&gt; 下载 Python 3.4。</String>
+  <String Id="FailureWS2K8R2MissingSP1">安装 [WixBundleName] 需要 Windows Server 2008 R2 Service Pack 1 及所有适用更新。
+
+请 &lt;a href="https://www.bing.com/search?q=how%20to%20install%20windows%20server%202008%20r2%20service%20pack%201"&gt;更新计算机&lt;/a&gt; 后重新安装。</String>
+  <String Id="FailureWS2K8MissingSP2">安装 [WixBundleName] 需要 Windows Server 2008 Service Pack 2 及所有适用更新。
+
+请 &lt;a href="https://www.bing.com/search?q=how%20to%20install%20windows%20server%202008%20service%20pack%202"&gt;更新计算机&lt;/a&gt; 后重新安装。</String>
+  <String Id="FailureWS2K3OrEarlier">安装和使用 [WixBundleName] 需要 Windows Server 2008 SP2 或更高版本。
+
+请访问 &lt;a href="https://www.python.org/"&gt;python.org&lt;/a&gt; 下载 Python 3.4。</String>
   <String Id="SuccessMaxPathButton">禁用路径长度限制</String>
   <String Id="SuccessMaxPathButtonNote">修改计算机配置，允许包括 Python 在内的程序绕过 260 字符的“MAX_PATH”限制。</String>
 
